### ISSUE TITLE
pandoc does not render greek symbols in ePub?

### ISSUE THREAD

**Issue by suknat** [jgm/pandoc#3231]
I'm writing in pandoc markdown with the intent to convert to ePub. I've been using standard HTML/MD to render greek symbols, like so: `&alpha;`. I believe pandoc supported exporting this to ePub a few months ago, but it seems broken now? I might be imagining this, but I have an entry in my diary suggesting it was working at some point! Things like `&deg;` are definitely working, so it would seem odd that the greek letters are not! Everything works fine when exporting to HTML / PDF. 

I can get around this by using `$\alpha$` in the text and `--webtex` at compile but am trying to avoid this as the rendered images seem very low res, and it affects readability. 

Is there a way to improve the resolution of the webtex images and/or write in html syntax? 

Improving the resolution of the webtex images would be particularly useful as I have math formulae written in LaTeX that are not very legible due to pixelation. 

I'm using iBooks to preview my ePub output, so happy to be corrected re webtex if this is an iBooks fault.

Thanks

**Comment by jkr** [comment]
Just doing a simple `echo '&alpha;' | pandoc -o out.epub`, it seems to convert fine. What version of pandoc are you using? (You can unzip the epub and look at the html pages to see if it's a conversion problem or an iBooks problem.)

**Comment by jgm** [comment]
This could be a font issue -- maybe the font you're using
lacks a glyph for that Greek letter?

Re webtex:  I don't know if there are ways to increase
pixelization.  However, there's a very slick method
using MathJax-node, which could allow all the math to
be replaced by SVG:
https://github.com/jgm/pandoc/issues/3153

Currently it would require a bit of postprocessing, but
we're thinking of making it easier to integrate with pandoc.

By the way, if you're targeting iBooks specifically,
I believe it supports epub3 with mathml.  So you could
try using pandoc to generate epub3 (it should automatically
use mathml for the math, I think, but it wouldn't hurt to
add --mathml).

+++ suknat [Nov 14 16 08:18 ]:

**Comment by suknat** [comment]
Thanks for both your comments and suggestions. 

I checked pmd version and it was 1.16 which I have now moved to 1.18. I can now report that the glyphs *are* being produced. Yay!

Re +jgm's suggestions:
- iBooks *does* handle uPub3 but some of the Math is not beautiful. For example, `$\bar{x}$` is reproduced with the bar a long distance above x. So long that you might miss it if you weren't looking for it!
- Use MathJax-node, I have now installed MathJax-node and await progress on the integration with pandoc (since my goal is ePub). 

Thanks!
